src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/matching_indices_list/matching_indices_list.html

```diff
@@ -44,6 +44,13 @@
           on-page-previous="matchingIndicesList.onPagePrevious"
           ng-if="matchingIndicesList.hasMultiplePages()"
         ></tool-bar-pager-buttons>
+
+        <span
+          ng-if="matchingIndicesList.hasMultiplePages()"
+          className="kuiSubduedText"
```

Comment:
You've been living in React-land for too long my friend. This should be `class`, not `className`. 😄

Reply:
hahaha oh man nice find